README.md

```diff
@@ -11,8 +11,7 @@ Steps to add additional rules:
 `kubectl apply -f codewind-odoclusterrole.yaml`
 `kubectl apply -f codewind-odoclusterrolebinding.yaml`
 
-
-Install ODO extension manually
+## Install Codewind ODO extension manually
 1. In your home directory, run the following command to clone the ODO extension repository
```

Comment:
In your home directory, run the following command to clone the ODO extension repository: